src/ApiService/ApiService/QueueSignalREvents.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Extensions.Logging;
+using System.Collections.Generic;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Microsoft.OneFuzz.Service.OneFuzzLib.Orm;
+using Azure.Storage.Queues.Models;
+using System.Linq;
+
+namespace Microsoft.OneFuzz.Service;
+
+public class QueueSignalREvents {
+    private readonly ILogger _logger;
+
+    public QueueSignalREvents(ILoggerFactory loggerFactory)
+    {
+        _logger = loggerFactory.CreateLogger<QueueSignalREvents>();
+    }
+
+    [Function("QueueSignalREvents")]
+    [SignalROutput(HubName="dashboard")]
+    public static string Run(
+        [QueueTrigger("signalr-events-refactored", Connection = "AzureWebJobsStorage")] string msg)
```

Comment:
I've been trying to avoid using the same queue names until we actually want to make the switch to the c# function. In some of the other refactored functions it might be ok because they'll fail (at least for now) but this one is fully functional so I think we'd be publishing duplicate events.

Reply:
fair enough. You will probably have to rename it when validating it in the check-pr eventually